--- libnsgif-0.2.0/src/lzw.c.orig	2017-10-13 10:55:06.000000000 +0200
+++ libnsgif-0.2.0/src/lzw.c	2017-10-30 11:35:43.430506835 +0100
@@ -168,7 +168,9 @@ static inline lzw_result lzw__next_code(
 		const uint8_t *data = ctx->sb_data + (ctx->sb_bit >> 3);
 		switch (byte_advance) {
 			case 2: code |= data[2] << 16;
+				/* Fall through */
 			case 1: code |= data[1] <<  8;
+				/* Fall through */
 			case 0: code |= data[0] <<  0;
 		}
 		ctx->sb_bit += code_size;
