--- libnsgif-0.1.0/Makefile~	2013-04-19 22:32:57.000000000 +0300
+++ libnsgif-0.1.0/Makefile	2013-04-27 15:35:27.323556353 +0300
@@ -30,6 +30,7 @@
 
 # Extra installation rules
 I := /include
+lib = lib
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):include/libnsgif.h
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib/pkgconfig:lib$(COMPONENT).pc.in
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib:$(OUTPUT)
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib)/pkgconfig:lib$(COMPONENT).pc.in
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib):$(OUTPUT)
