--- libnsgif-0.1.2/Makefile.orig	2015-03-16 12:32:37.351846552 +0100
+++ libnsgif-0.1.2/Makefile	2015-03-16 12:33:06.331689551 +0100
@@ -13,10 +13,7 @@ include $(NSSHARED)/makefiles/Makefile.t
 WARNFLAGS := -Wall -W -Wundef -Wpointer-arith -Wcast-align \
 	-Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes \
 	-Wmissing-declarations -Wnested-externs -pedantic
-# BeOS/Haiku standard library headers create warnings
-ifneq ($(BUILD),i586-pc-haiku)
-  WARNFLAGS := $(WARNFLAGS) -Werror
-endif
+
 CFLAGS := -D_BSD_SOURCE -I$(CURDIR)/include/ \
 	-I$(CURDIR)/src $(WARNFLAGS) $(CFLAGS)
 ifneq ($(GCCVER),2)
